samples/fabric/simpleassettransfer/tokenasset.go

```diff
@@ -387,19 +386,12 @@ func (s *SmartContract) GetTokenAssetClaimStatus(ctx contractapi.TransactionCont
 		return blankClaimBytes64, err
 	}
 	if claimAssetDetails == nil {
-		return blankClaimBytes64, err
+		// represents the scenario that the asset was not claimed by the remote network
+		return blankClaimBytes64, nil
 	}
 
 	// Validate returned asset details using app-specific-logic
-
-	// The asset type should be recorded, so we should look that up first
-	exists, err := s.TokenAssetTypeExists(ctx, assetType)
```

Comment:
I think we can retain this check, as once a tokenType is created its never deleted automatically. It's different from actual tokens, which can be spent. Also this leads to one more doubt, during ClaimRemoteToken, if tokentype doesn't exist, are we creating it? I'm wondering what will happen when we do house-token Transfer to fabric.

Reply:
As discussed, we assume that this token type was created separately beforehand only (and that's not in the control of asset transfer flow ideally). So, don't need to create it explicitly within this flow explicitly, if that didn't exist.